contracts/protocol/wrapper/VPoolWrapper.sol

```diff
@@ -172,6 +172,12 @@ contract VPoolWrapper is IVPoolWrapper, IUniswapV3MintCallback, IUniswapV3SwapCa
     }
 
     function setFundingRateOverride(int256 fundingRateOverrideX128_) external onlyGovernance {
+        uint256 fundingRateOverrideX128Abs = fundingRateOverrideX128_.absUint();
+        // ensure that funding rate magnitude is < 100% APR
+        if (
+            fundingRateOverrideX128_ != FUNDING_RATE_OVERRIDE_NULL_VALUE &&
+            fundingRateOverrideX128Abs > FixedPoint128.Q128 * 365 days
```

Comment:
Funding rate of `FixedPoint128.Q128` means 360000% per hour. 

What is our max limit we wanna keep, if its 100% per hour then it should be `(FixedPoint128.Q128 / 3600)`